$NetBSD: patch-Makefile.conf,v 1.4 2016/02/06 14:01:13 jaapb Exp $

Have BINDIR be determined from package Makefile, do not call git
Also determine tyxml and react usage based on options rather than detection
--- Makefile.conf.orig	2016-01-25 19:35:03.000000000 +0000
+++ Makefile.conf
@@ -1,6 +1,6 @@
 
 # Where binaries are installed:
-BINDIR := `dirname  \`which ocamlc\``
+#BINDIR := `dirname  \`which ocamlc\``
 
 ####
 
@@ -40,8 +40,8 @@ WITH_DERIVING ?= $(shell if [ -f `ocamlf
 
 WITH_GRAPHICS ?= $(shell if [ -f `ocamlfind query graphics 2> /dev/null`/graphics.cmi ]; then echo YES; else echo NO; fi)
 
-WITH_REACT ?= $(shell if [ -f `ocamlfind query reactiveData 2> /dev/null`/reactiveData.cma ]; then echo YES; else echo NO; fi)
-WITH_TYXML ?= $(shell if [ -f `ocamlfind query tyxml 2> /dev/null`/tyxml_f.cma ]; then echo YES; else echo NO; fi)
+WITH_REACT ?= ${USE_REACT}
+WITH_TYXML ?= ${USE_TYXML}
 
 WITH_FINDLIB?=YES
 
@@ -60,7 +60,7 @@ endif
 
 METAOCAML ?= 0
 
-VERSION_GIT := $(shell git log -n1 --pretty=format:%h 2> /dev/null)
+#VERSION_GIT := $(shell git log -n1 --pretty=format:%h 2> /dev/null)
 
 %.ml: %.cppo.ml
-	cppo $< -o $@ -V OCAML:`ocamlc -version`
\ No newline at end of file
+	cppo $< -o $@ -V OCAML:`ocamlc -version`
